android/app/src/main/java/com/textile/textilenode/TextileNode.java

```diff
@@ -289,7 +293,8 @@ public void pairDevice (String pkb64, Promise promise) {
     public void getFilePath(String uriString, Promise promise) {
         Uri uri = Uri.parse(uriString);
         try {
-            promise.resolve(RealPathUtil.getRealPath(reactContext, uri));
+            String path = RealPathUtil.getRealPath(reactContext, uri);
+            promise.resolve(path);
```

Comment:
Noticed some weirdness here. Seems to work fine on a real device, but on the emulator, the path returned here eventually is identified as 'file doesn't exist'. Anyone seen that before?

Reply:
@andrewxhill ^?

Reply:
haven't seen it, no... android only?

Reply:
Yea android only.